### PR TITLE
Partitioner: adapt the tree and the overview tabs

### PR DESCRIPTION
**Pull request by ancorgs**
This is part of the on-going process to reorganize the Partitioner UI into the branch `partitioner-ui-02`.

This pull request:

  - Adjusts the nesting and the labels of the left tree
  - Changes the overview tabs for all types of devices
  - Removes headings that now look redundant

new_overview

Unit tests adapted.

### CODE REVIEW

**Comment by ancorgs** [comment]
The test coverage decrease is caused by the reduction of the total amount of code in places like the overview tree (that has fewer branches) and the titles/icons of several pages.

File: src/lib/y2partitioner/widgets/overview_tab.rb [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  module Widgets
+    # Base class to represent the tab that offers an overview of a device (eg. a disk)
+    # and its subdevices (eg. its partitions and/or Btrfs subvolumes)
+    class OverviewTab < CWM::Tab
+      # @return [Y2Storage::Device]
+      attr_reader :device
+
+      # Constructor
+      #
+      # @param device [Y2Storage::BlkDevice]
```
Comment (joseivanlopez) [inline src/lib/y2partitioner/widgets/overview_tab.rb:35]: NP: only BlkDevice? This is also used for VGs, right?

**Review by joseivanlopez — COMMENTED** [review]
If I am not wrong, *widgets/partititons_tab.rb* can be removed now, right?

**Review by joseivanlopez — APPROVED** [review]
LGTM

**Pull request merged**